pigeon/pigeon/urls.py

```diff
@@ -33,11 +33,12 @@
 from django.conf.urls import include, url
 from django.contrib import admin
 from django.conf import settings
+from django.conf.urls.i18n import i18n_patterns
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
-    url(r'', include('weights.urls')),
 ]
+urlpatterns += i18n_patterns(url(r'',  include('weights.urls')))
```

Comment:
Will it leave unmodified SSO callback urls (my guess is no and we should have put them else where :p)